src/containers/trends/TrendsContainer.js

```diff
@@ -199,6 +199,9 @@ export class TrendsContainer extends React.Component {
       mostRecent: mostRecent.toISOString(),
       xScale: scaleLinear().domain([0, 864e5]),
       yScale,
+      selectDay: (date) => {
+        this.props.onSelectDay(datetime.midDayForDate(date, timePrefs));
+      },
```

Comment:
Why does this need to be in state? Can't we just define it as an instance method?